android/sdk/src/main/java/com/taobao/weex/utils/WXSoInstallMgrSdk.java

```diff
@@ -304,6 +307,61 @@ public static void copyStartUpSo() {
     }
   }
 
+  public static void copyJssRuntimeSo(){
+    Log.e("test->", "enter copyJssRuntimeSo: ");
```

Comment:
Please remove `Log.e` or change them to WXLogUtils.e

Reply:
done